docs/faq/downstream_analysis.ipynb

```diff
@@ -0,0 +1,85 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": [
+     "active-ipynb",
+     "remove-input",
+     "remove-output"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "try:\n",
+    "    import openmdao.api as om\n",
+    "    import dymos as dm\n",
+    "except ImportError:\n",
+    "    !python -m pip install openmdao[notebooks]\n",
+    "    !python -m pip install dymos\n",
+    "    import openmdao.api as om\n",
+    "    import dymos as dm"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# How do connect the outputs of Dymos to a downstream analysis?\n",
+    "\n",
+    "One of the design goals of Dymos is to allow the trajectory to be a part of a larger multidisciplinary optimization problem.\n",
+    "Sometimes, you may want to take the results from the Dymos trajectory and feed them to some downstream analysis.\n",
+    "\n",
+    "In the case of only being concerned with the final value of some parameter, this can be accomplished by connecting the relevant output from\n",
+    "the timeseries with `src_indices=[-1]`.\n",
+    "\n",
+    "For example, something like the following might be used to connect the final value of the state `range` to some downstream component.\n",
+    "\n",
+    "```python\n",
+    "problem.model.connect('trajectory.phase0.timeseries.states:range',\n",
+    "                      'postprocess.final_range',\n",
+    "                      src_indices=[-1])\n",
+    "```\n",
+    "\n",
+    "```{Note}\n",
+    "    We _highly_ recommend you use the phase timeseries outputs to retrieve outputs from Dymos, since it is transcription-indepdendent.\n",
+    "```\n",
```

Comment:
For some reason, _highly_ renders with the underscore instead of italic.  I think you need to remove the indentation to fix it.